css-structure_taco-truck/css/card.css

```diff
@@ -0,0 +1,24 @@
+.main-content__menu-list {
```

Comment:
Because it using BEM it would be better to have `main-content__menu-list` in `main.css`. 
All classes that start with `main-content` is better to have them in the same file.